core/trino-main/src/main/java/io/trino/execution/SqlTaskManager.java

```diff
@@ -84,7 +84,7 @@
 import static java.util.concurrent.Executors.newScheduledThreadPool;
 
 public class SqlTaskManager
```

Comment:
Should we rename this to `TaskManager` to minimize other changes?

Make `failAbandonedTasks()` private since it's not meant to be used externally.

Reply:
> Should we rename this to TaskManager to minimize other changes?

Actually (if done in a single commit) it will be recorded by diff as a bigger change, as `TaskManager.java` already exists; it will be treated as an edit of `TaskManager.java`, not a rename-with-changes of `SqlTaskManager.java` to `TaskManager.java`

Reply:
> Make failAbandonedTasks() private since it's not meant to be used externally.

done